### PR TITLE
[doc] Improve s3 operator example by adding task upload_keys

### PR DESCRIPTION
**Pull request by zhongjiajie**
for now, our s3 bucket example does without declaring task `upload_keys` but we set dependence with it. I think we should better add `upload_keys` declare to the example for more sence.
image

### CODE REVIEW

**Comment by ferruzzi** [comment]
I can see what you are saying, yeah.  I'm working on planning a bit of a project to get a lot of the AWS examples and docs up to par, so there should be a lot of improvements in that area in the near future.  I'll keep cases like this in mind.

**Comment by zhongjiajie** [comment]
@ferruzzi So should we merge or should we just close this issue?

**Comment by ferruzzi** [comment]
@zhongjiajie I say go ahead, it's non-breaking either way and I think we're going to focus on the ones that don't have any documentation or sample DAGs first so I don't see a reason to block this just because we might update it later.  I'm not a committer but you have my non-binding approval :P

**Review by potiuk — APPROVED** [review]

**Comment by zhongjiajie** [comment]
Thanks~

**Pull request merged**
